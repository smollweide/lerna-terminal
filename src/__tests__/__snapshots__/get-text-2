%7B%22text%22:%22%5Cu001b%5B31merror%200%5Cu001b%5B39m%22%7D