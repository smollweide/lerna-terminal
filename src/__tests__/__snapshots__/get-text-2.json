{"text":"\u001b[31merror 0\u001b[39m"}